automation/test.sh

```diff
@@ -68,14 +78,17 @@ data:
 ---
 EOF
 
-key="/tmp/secrets/accessKeyId"
-token="/tmp/secrets/secretKey"
+if [ "${KUBE_CMD}" == "oc" ]
```

Comment:
You don't have to put this block of code inside another if statement

Reply:
Removed it out of the block